reference/stories/primitive-types.rna-and-dna-processing.md

```diff
@@ -0,0 +1,7 @@
+# RNA & DNA Processing
+
+In biology libraries, Nucleic Acids may be encoded as 4-bit types (see for example [BioSymbols.jl](https://github.com/BioJulia/BioSymbols.jl/blob/master/src/nucleicacid.jl)).
+This can be used as a context for an exercise introducing primitive types, or other concepts related to bit manipulation.
+
+The v2 exercises hamming, rna-transcription and nucleotide-count could be reused but with primitive types instead of strings/chars.
```

Comment:
```suggestion
The v2 exercises `hamming`, `rna-transcription` and `nucleotide-count` could be reused but with primitive types instead of `strings`/`chars`.
```

Perhaps also link to [`types/char.md`](https://github.com/exercism/v3/blob/stories/bio/reference/types/char.md) and [`types/string`](https://github.com/exercism/v3/blob/stories/bio/reference/types/string.md)?